wp-content/themes/fundawande/templates/lms/template-coach-dash.twig

```diff
@@ -11,57 +11,72 @@
 
 {# Twig block to be included in base.twig #}
 {% block content %}
-    <div class="wrapper coach-dash-wrapper" id="lms-page-wrapper" xmlns="http://www.w3.org/1999/html">
+    <div class="wrapper coach-dash-wrapper" id="page-wrapper" xmlns="http://www.w3.org/1999/html">
 
         {# Sign up information band #}
         <section class="container ">
             <div class="row">
                 <div class="col-12">
-                    <h1 class="lbreaker-lms-purple mt-4">Coach dashboard</h1>
+                    <h1 class="lbreaker-lms-purple mt-4">Coach assessment review dashboard</h1>
+                    <a class="primary-btn-outline px-1 py-1 rounded" href="/coach-progress">Go to progress dashboard</a>
+
                 </div>
                 <div class="col-12 mt-4">
-                    <div class="row">
-                        <div class="col-auto px-3 mb-4 mb-md-0">
-                            <select type="select" class="form-control custom-select coach-filter"   id="courseSelect"  style="max-width:120px" placeholder="" value="" required>
-                                <option value=""  selected >All course</option>
-                                {% for course in courses %}
-                                    <option value="{{course.ID }}" {% if selected_course ==  course.ID %}selected{% endif %}  >{{course.title}}</option>
-                                {% endfor %}
-                            </select>
-                        </div>
-                        <div class="col-auto px-3 ">
-                            <select type="select" class="form-control custom-select coach-filter " id="coachSelect" style="max-width:120px"  placeholder="" value="" required>
-                                <option value="" disabled selected >Filter by coach</option>
-                                <option value=""  >All coaches</option>
-                                <option diabled  value="" >No coaches available yet</option>
+                    <form id="" action=""  method="get">
+                        <div class="row">
+                        
+
+                            <div class="col-auto px-3 mb-4 mb-md-0">
+                                <select type="select" name="fw_course" class="form-control custom-select coach-filter"   id="courseSelect"  style="max-width:150px" placeholder="" value="" required>
+                                    <option value="" disabled  selected >Choose a course</option>
+                                    {% for course in courses %}
+                                        <option value="{{course.ID }}" {% if selected_course ==  course.ID %}selected{% endif %}  >{{course.post_title}}</option>
+                                    {% endfor %}
+                                </select>
+                            </div>
+                            <div class="col-auto px-3 ">
```

Comment:
If I understand the function of this filter correctly, it filters learners based on who their assigned coach is? Perhaps its a filter that should only be visible if the user accessing the coach dashboard is an administrator?